tasks/main.yml

```diff
@@ -1,33 +1,6 @@
 ---
 - include_tasks: "install-{{ ansible_facts.os_family }}.yml"
 
-# Setup NTP
-- name: Disable timesyncd
-  shell: timedatectl set-ntp no
```

Comment:
Does RHEL / CentOS not use `timedatectl`?

Reply:
It does. I grabbed that by mistake since it was under the `# Setup NTP` section.